README.md

```
@@ -236,18 +238,20 @@ ollama pull huihui_ai/dolphin3-abliterated:8b
 #### For 16GB VRAM (Recommended)
 ```bash
 # Balanced quality and speed
-ollama pull huihui_ai/dolphin3-abliterated:8b     # Fast, versatile
-ollama pull vanilj/mistral-nemo-12b-celeste-v1.9:Q8_0  # Creative writing
+ollama pull huihui_ai/dolphin3-abliterated:8b         # Fast, versatile
+ollama pull vanilj/mistral-nemo-12b-celeste-v1.9:Q8_0 # Creative writing
 ```
 
-#### For 24GB VRAM (Premium)
+#### For 24GB VRAM (Premium - February 2026 Update)
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)